src/contract/method.rs

```diff
@@ -262,7 +262,11 @@ impl<T: Transport, R: Detokenize> CallFuture<T, R> {
                         from: builder.m.tx.from.map(|account| account.address()),
                         to: builder.m.tx.to.unwrap_or_default(),
                         gas: builder.m.tx.gas,
-                        gas_price: builder.m.tx.gas_price,
+                        gas_price: builder
+                            .m
```

Comment:
What is dot m?

Reply:
It’s the method builder. The `ViewMehodBuilder` is basically a helper type that just wraps an inner `MethodBuilder` but makes it so there are less things you can call (it doesn’t let you call `send` for example).